frontend/src/app/components/MainInstallButton/index.js

```diff
@@ -74,14 +96,15 @@ export default class MainInstallButton extends Component<MainInstallButtonProps,
       <LayoutWrapper flexModifier={layout} helperClass="main-install">
         <div className="main-install__spacer" />
         {/* $FlowFixMe */}
-        {(isMinFirefox && !isMobile) ? this.renderInstallButton(isInstalling, hasAddon) : this.renderAltButton(isFirefox, isMobile) }
+        {(showWebButton) ? this.renderWebExperimentButton() : (isMinFirefox && !isMobile) ? this.renderInstallButton(isInstalling, hasAddon) : this.renderAltButton(isFirefox, isMobile)
```

Comment:
Yeah, this is getting hairy enough to break out into a more verbose if/then/else in like a separate `renderMainButton` function:
```
if (showWebButton) {
...
} else if (isMinFirefox && !isMobile) {
...
} else {
...
}
```